patches/lede/0086-ar71xx-set-status-led-for-the-gl-boards.patch

```diff
@@ -318,7 +318,7 @@ index 42f4415d7fe0252aadf39e2ca50f96566c023728..412c562fa042e7abb0ccb35208bb5582
  static struct gpio_led gl_mifi_leds_gpio[] __initdata = {
  	{
 -		.name = "gl-mifi:wan",
-+		.name = "gl-mifi:greeen:wan",
++		.name = "gl-mifi:green:wan",
```

Comment:
Are you modifying a patch that was pulled from OpenWrt? If so, why didn't you submit it upstream instead?

Reply:
It's in fact updated in OpenWRT (also in the 18.06 release).

https://github.com/openwrt/openwrt/commit/58fe5279318b0bcc398ea5bde68db4ce204daa8b

So we should backport this patch instead of fiddling in existing patches.

Reply:
the pull request for the mistype is just a correction of an active lede patch in order to get wlan led working. To keep it stupid simple is would suggest to update 2018.1.x branch lede patch with correction like i did in my source that's currently compiling for 8 Domains in Rhein-Sieg

Reply:
the pull request regarding target/ar71-generic is valid for 2018.1 and master. lede and openwrt both support gl-mifi .
gluon-2018.1.x based firmware is in productive test for more than 24 hours now:
https://map.freifunk-siegburg.de/#!v:m;n:e4956e44b048